hexrd/wppf/peakfunctions.py

```diff
@@ -417,9 +403,9 @@ def _gaussian_pink_beam(alpha,
     t2 = erfc(z)
     g = np.zeros(tth_list.shape)
     zmask = np.abs(del_tth) > 5.0 
```

Comment:
Looks like this `zmask` isn't actually used anymore? We can remove it if it is unused.

Reply:
Good catch. I think using the `zmask` makes the code a bit better behaved so I'll put it back.